### PR TITLE
build(query-router): Deploy Query Routing

### PR DESCRIPTION
**Pull request by luke-biel**

### CODE REVIEW

**Review by esavier — COMMENTED** [review]
proper naming of PR or drafts, please

**Comment by luke-biel** [comment]
We don't have standard naming

File: examples/deploy/docker-compose.yml [hunk 1]
```diff
@@ -57,14 +57,21 @@ services:
       - ./docker-volume/postgres1/db:/var/lib/postgresql/data
       - ./docker-volume/postgres1/init:/docker-entrypoint-initdb.d
 
+  victoria_metrics1:
```
[the remark this answers is not in the document]
Comment (USSAMOH2) [reply examples/deploy/docker-compose.yml:60]: Also, do we need the index after each service?

File: helm/cdl/values-local.yaml [hunk 2]
```diff
@@ -35,16 +35,21 @@ documentStorage:
...
+    replicaCount: 1
+  queryService:
+    image: cdl-query-service
+    bin: query-service
+    inputPort: 50102
+    replicaCount: 1
 
 sled-document:
-  replicaCommandCount: 1
+  replicaCommandCount: 0
```
Comment (USSAMOH2) [inline helm/cdl/values-local.yaml:52]: Why are the replica counts 0?

File: build.sh [hunk 3]
```diff
@@ -7,17 +7,19 @@ set -ex
 # CDL_VERSION=0.1.0
 # CDL_PUSH=true
```
Comment (USSAMOH2) [inline build.sh:8]: Consider renaming this to something more readable (e.g. PUSH_TO_DOCKER_HUB or something similar).

**Review by USSAMOH2 — CHANGES_REQUESTED** [review]

**Comment by luke-biel** [comment]
Changes in this were partially merged. Helm still needs to be updated, I'll do that in separate one.

**Pull request closed without merging**